src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -669,4 +669,32 @@ public static String expandAuthorInitials(String name) {
         return sb.toString().trim();
     }
 
+    /**
+     * Return a String with n spaces
+     *
+     * @param n Number of spaces
+     * @return String with n spaces
+     */
+    public static String nSpaces(int n) {
+        return nCharacters(n, ' ');
+    }
+
+    /**
+     * Return a String with n copies of the char c
+     *
+     * @param n Number of copies
+     * @param c char to copy
+     * @return String with n copies of c
+     */
+    public static String nCharacters(int n, char c) {
```

Comment:
Such a method already exists in Guava: http://docs.guava-libraries.googlecode.com/git-history/v10.0.1/javadoc/com/google/common/base/Strings.html#repeat%28java.lang.String,%20int%29
Should we use this method instead?
If not I would propose to change the name of `nCharacters` to `repeat(char, int)` and similarly `repeatSpaces(int)`